packages/coding-standards/README.md

```diff
@@ -25,22 +25,22 @@ php composer require shopsys/coding-standards
 
 ## Usage
 
-Create `custom-coding-standard.neon` config file in your project which includes predefined rulesets. 
+Create `custom-coding-standard.yml` config file in your project which includes predefined rulesets. 
 You can also [customize the rules](./UPGRADE.md#version-4.0-and-higher) 
 and even add your own sniffs and fixers in the config.
 
-```neon
-#custom-coding-standard.neon
+```yaml
```

Comment:
Shouldn't content looks like this?

```yaml
#custom-coding-standard.yml
imports:
    - { resource: 'vendor/symplify/easy-coding-standard/config/psr2.yml' }
    - { resource: 'vendor/shopsys/coding-standards/shopsys-coding-standard.yml' }
```

Well, I'm not sure how `yml` works, so thats why I ask :-)

Reply:
@simara-svatopluk Here, it should.

`%vendor_dir%` is just variable to make it usable in case of different location, see [Symplify PR](https://github.com/Symplify/Symplify/pull/741))